README.md

```diff
@@ -1,17 +1,17 @@
-# Terraform Module Template
+# Terraform MySQL Module
```

Comment:
Update top-level header according to official "display name" for this service ([ref](https://azure.microsoft.com/en-us/products/mysql)):

```suggestion
# Azure Database for MySQL Terraform module
```